src/core/zowe/core_for_zowe_sdk/config_file.py

```diff
@@ -213,6 +212,7 @@ def get_profile(
             self.init_from_file(validate_schema)
 
         if profile_name is None and profile_type is None:
+            self.__logger.error(f"Failed to load profile '{profile_name}' because Could not find profile as both profile_name and profile_type is not set")
```

Comment:
Can we adjust this error for clarity? It looks like we are passing in the `profile_name` for the error that's logged, but `profile_name` will always be `None` if this condition is satisfied.

```suggestion
            self.__logger.error(f"Failed to load profile: profile_name and profile_type were not provided.")
```